vpr/src/base/echo_files.cpp

```diff
@@ -62,8 +63,8 @@ char* getEchoFileName(enum e_echo_files echo_option) {
 }
 
 void alloc_and_load_echo_file_info() {
-    echoFileEnabled = (bool*)vtr::calloc((int)E_ECHO_END_TOKEN, sizeof(bool));
-    echoFileNames = (char**)vtr::calloc((int)E_ECHO_END_TOKEN, sizeof(char*));
+    echoFileEnabled = new bool[(int)E_ECHO_END_TOKEN]();
```

Comment:
I think we should explicitly initialize these variables to false & NULL.  Looking up info on when C++ initializes built-in types like this to 0 when you use new gives complex answers: e.g. https://stackoverflow.com/questions/15212261/default-initialization-of-pod-types-in-c
says sometimes they are initialized to 0, sometimes not, depending on the exact syntax of the new call.  Not sure if that's correct or not, but since we rely on these being initialized to false/NULL we should make that explicit.